flashcards/backend/decks/factories.py

```diff
@@ -1,5 +1,6 @@
 from factory import Faker, Sequence, SubFactory
 from factory.django import DjangoModelFactory, ImageField
+import factory
```

Comment:
Why import whole factory?

Reply:
just automatic import
i've fixed. thanks for checking